src/App.js

```diff
@@ -1,24 +1,55 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Routes, Route, NavLink } from 'react-router-dom';
 import './App.css';
 
+function List() {
+  return (
+    <>
+      <h1>List of Items</h1>
+    </>
+  );
+}
+
+function AddAnItem() {
+  return (
+    <>
+      <h1>Add an Item</h1>
+    </>
+  );
+}
+
+function NavLinks() {
+  return (
+    <nav style={{ position: 'fixed', bottom: 0 }}>
+      <NavLink
+        to="/"
+        style={({ isActive }) =>
+          isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
+        }
+      >
+        List of Items
+      </NavLink>
+      <NavLink
+        to="/add-an-item"
+        style={({ isActive }) =>
```

Comment:
Noice! I didn't know that `NavLink`'s `style` prop can take a callback like this! 

It looks like it [also takes an `activeStyle`](https://v5.reactrouter.com/web/api/NavLink/activestyle-object) too. So cool!